client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/style.scss

```diff
@@ -20,13 +20,31 @@
 	@extend .notification-settings-title-light;
 	position: absolute;
 	inset-block-start: 50px;
-
 }
 
 .notification-settings__content {
 	margin-block-start: -16px;
 }
 
+.notification-settings__content-heading-with-beta {
+	display: flex;
+	align-items: center;
+	gap: 5px;
```

Comment:
As per the FIGMA design, this should be 8px

<img width="267" alt="Screenshot 2023-05-22 at 11 01 11 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/80d35f15-5984-44a8-9b4e-05e04cb65e93">